app/services/cross_strategy_coordinator.py

```
@@ -0,0 +1,570 @@
+                    available_capacity = max_allowed - current_exposure
+                    
+                    if available_capacity > 0.01:  # At least 1% available
+                        modified_signal = coordination.signal.copy()
+                        modified_signal["position_size_pct"] = available_capacity * 100
+                        
+                        coordination.coordination_action = "modify"
+                        coordination.modified_signal = modified_signal
+                        coordination.conflict_resolution = f"Reduced position size to {available_capacity*100:.1f}% due to concentration limits"
+                        
+                        self.logger.info(
+                            f"📏 Position size reduced for concentration management",
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)